lepidopter-fh/opt/ooni/lepidopter-update/updater.py

```diff
@@ -0,0 +1,357 @@
+#!/usr/bin/env python2
+
```

Comment:
We should probably add a comment in here mentioning what is the version of the updater (i.e. what is the latest commit hash it was copied from) and where the source lives.